azurelinuxagent/common/osutil/ubuntu.py

```diff
@@ -94,6 +94,22 @@ def __init__(self):
         super(Ubuntu18OSUtil, self).__init__()
         self.service_name = self.get_service_name()
 
+    def restart_if(self, ifname, retries=3, wait=5):
+        """
+        Restart systemd-networkd
+        """
+        retry_limit=retries+1
+        for attempt in range(1, retry_limit):
+            return_code=shellutil.run("systemctl restart systemd-networkd")
```

Comment:
@taoyama sorry for the delay with this PR. everything looks, just one request: the use of shellutil.run is deprecated; instead we use shellutil.run_command (you can see an instance of its usage in the implementation of this same method in the default os class). thanks

Reply:
Thanks for your review @narrieta .
Though I've found  shellutil.run in other place I leave that for now and replaced only here.